addons/mail/static/src/models/thread_viewer/thread_viewer.js

```diff
@@ -95,6 +95,20 @@ function factory(dependencies) {
         // Private
         //----------------------------------------------------------------------
 
+        /**
+         * @private
+         * @returns {boolean}
+         */
+        _computeIsLoading() {
+            if (
+                this.thread && this.threadCache && this.threadCache.isLoading &&
+                !this.isShowingLoading && !this._isPreparingLoading
```

Comment:
I think it was to allow to be updated without using `.update` method as it was changed in updateAfter and I wonder if infinite loop were not possible. Now that's no more the case, I suppose it can be a field